PendingReleaseNotes

```diff
@@ -77,3 +77,7 @@
   librbd to load the plugin by adding the following to your configuration::
 
     rbd_plugins = parent_cache
+
+* Monitors now have a config option ``mon_osd_warn_num_repaired``, 10 by default.
+  If any OSD has more than this many repaired reads a ``OSD_TOO_MANY_REPAIRS``
```

Comment:
not sure a user will understand what a 'repaired read' is in this context - maybe something like 'If any OSD has repaired more than this many checksum errors in stored data'...

Reply:
@jdurgin Do we really want to get specific and mention checksum errors?  For filestore it would be just an EIO from the disk driver, for example.

Reply:
maybe 'I/O errors' then